integration_tests/src/main/python/cache_test.py

```diff
@@ -23,17 +23,21 @@
 from generate_expr_test import four_op_df
 from marks import incompat, allow_non_gpu, ignore_order
 
+enableVectorizedConf = [{"spark.sql.inMemoryColumnarStorage.enableVectorizedReader" : "true"},
```

Comment:
Why exactly are we testing with this on/off if we are not using the config as a part of the cache code?  The main reason for my concern is that these are some of the tests that take the most time to run, and if we double the runtime with little to no extra coverage I am not sure it is worth it.

Reply:
Here's why I think its needed

In case RapidsConf.isSqlEnabled we are always reading columnar so the conf `spark.sql.inMemoryColumnarStorage.enableVectorizedReader` is ignored 

In case RapidsConf.isSqlEnabled is false, we will depend on the Serializer's `supportsColumnarOutput` which will always return true in our tests because all of our datatypes in cache_test.py are of `AtomicType`. So we will never hit the case where we read non-columnar. In other words `convertCachedBatchToInternalRow` will never be called. If we always set it to false only then the part of code, not implemented yet, where we read the CachedBatch directly to ColumnarBatch will never be tested. 

Do you think I am right?

Reply:
Oh sorry.  You are right.  I mixed that config up with a similar one for parquet and it confused the heck out of me.